lib/msal-common/src/telemetry/performance/PerformanceEvent.ts

```diff
@@ -101,25 +101,25 @@ export enum PerformanceEvents {
     RefreshTokenClientExecuteTokenRequest = "refreshTokenClientExecuteTokenRequest",
 
     /**
-     * Time spent creating default headers for requests to token endpoint
+     * Time taken for acquiring refresh token , records RT size
      */
-    BaseClientCreateTokenRequestHeaders = "baseClientCreateTokenRequestHeaders",
+    RefreshTokenClientAcquireToken = "refreshTokenClientAcquireToken",
 
     /**
-     * Used to measure the time taken for completing embedded-broker handshake (PW-Broker).
+     * Time taken for acquiring cached refresh token 
      */
-    BrokerHandhshake = "brokerHandshake",
```

Comment:
Reason for removing this?

Reply:
These two events are not being used in the PW-Broker flow code in 1p . Were added initially prior to the actual instrumentation.

Reply:
That seems unrelated to the rest of this PR, please open a separate PR if so.